daemon/cmd/status.go

```diff
@@ -812,6 +813,41 @@ func (d *Daemon) startStatusCollector(cleaner *daemonCleanup) {
 				}
 			},
 		},
+		{
+			Name: "config-settings",
```

Comment:
I think we should set an `Interval` field here so that this code doesn't run every on the default of 5s. I think the default of 5s is too frequent to be hitting the apiserver fetching the configmap and the CNC resources. I would start at 1m for now.